src/spikeinterface/core/template.py

```diff
@@ -24,6 +25,8 @@ class Templates:
         Array of channel IDs. If `None`, defaults to an array of increasing integers.
     unit_ids : np.ndarray, optional default: None
         Array of unit IDs. If `None`, defaults to an array of increasing integers.
+    probe: Probe , optional default: None
```

Comment:
```suggestion
    probe: Probe, default: None
```

Reply:
can this be a `ProbeGroup` too?

Reply:
No. This is too complicated. A template is Probe only.